index.js

```diff
@@ -1,4 +1,4 @@
-'use strict'
+'use strict';
```

Comment:
The project does not use semicolons. I'd recommend that the PR adhere to that as well instead of also changing the style. Also, the introduction of semicolons is not consistent, as it is missing multiple places.